R/combined.R

```diff
@@ -286,6 +297,55 @@ combined_aggregate_data <- function(data) {
 }
 
 
+combined_aggregate_severity <- function(severity, samples) {
+
+  # We first need to sum admitted_inc and diagnoses_inc trajectories to get
+  # all_admissions_inc, which we'll use as weight to aggregate hospital severity
+  for (i in names(samples)) {
+    admitted_inc <- samples[[i]]$trajectories$state["admitted_inc", , ]
+    diagnoses_inc <- samples[[i]]$trajectories$state["diagnoses_inc", , ]
+    all_admissions <- array(admitted_inc + diagnoses_inc, dim = c(1, 10, 744))
```

Comment:
I assume you don't want the dim hardcoded like this?

Reply:
Probably best to add `drop = FALSE` in the above two lines and then can just have `all_admissions <- admitted_inc + diagnoses_inc`

Reply:
makes sense... thanks!